src/steam-user/steam-user.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { SteamUserService } from './steam-user.service';
+import { CreateSteamUserDto } from './dto/create-steam-user.dto';
+import { UpdateSteamUserDto } from './dto/update-steam-user.dto';
+
+@Controller('steam')
```

Comment:
What do you think about `users/steam`?
```suggestion
@Controller('users/:id/steam')
```

It should act like a connected Steam account to a User with `id={:id}`